pkg/gotohelm/helmette/sprig.go

```diff
@@ -38,8 +39,12 @@ func Keys[K comparable, V any](m map[K]V) []K {
 }
 
 // Merge is a go equivalent of sprig's `merge`.
-func Merge[K comparable, V any](dst, src map[K]V) map[K]V {
-	maps.Copy(dst, src)
+func Merge[K comparable, V any](dst map[K]V, sources ...map[K]V) map[K]V {
```

Comment:
Could we also add in a test case or two to the sprig test package in gotohelm to ensure that sprig works the same way as the transpiled version?

Reply:
Ok, I will add that.